composer.json

```
@@ -163,7 +163,8 @@
         "laminas/laminas-code": "^4.0",
         "symfony/doctrine-bridge": "4.4.16",
         "symfony/property-info": "4.4.22 || 5.2.7",
-        "symfony/serializer": "4.4.19 || 5.2.2"
+        "symfony/serializer": "4.4.19 || 5.2.2",
+        "symfony/flex": "^v1.14.0"
```

```suggestion
        "symfony/flex": "^1.14.0"
```
Wrong order of libraries

---

seems like this conflict wasnt needed to make the build green, github actions only breaks with it